server-phoenix/lib/helios/slack_channel_names.ex

```diff
@@ -14,4 +14,8 @@ defmodule Helios.SlackChannelNames do
   def with_id(query \\ __MODULE__, channel_id) do
     from(q in query, where: q.channel_id == ^channel_id, select: {q.channel_id})
   end
+
+  def name_to_id(query, channel_id) do
```

Comment:
this seems confusing, are we taking an id and returning a name? might make more sense to name this `id_to_name`

Reply:
... or could also name it name_from_id 🤷